Makefile

```diff
@@ -102,7 +102,7 @@ ARCH := $(shell echo $(ARCHES) | awk -F' ' '{ for(i=1;i<=NF;i++) print "-arch "
 
 ifeq ($(UNAME),Linux)
 $(warning Building on Linux)
-SYSROOT         ?= $(HOME)/cctools/SDK/iPhoneOS13.2.sdk
+TARGET_SYSROOT         ?= $(HOME)/cctools/SDK/iPhoneOS13.2.sdk
```

Comment:
Please fix the formatting here. See the above line. You’ll know what I mean. Same with any other instance of this happening.